apps/e2e-test/selector-mappings/components/settings.yaml

```diff
@@ -0,0 +1,34 @@
+setting button: '[test-id=settingsButton]'
+settings drawer: '//*[@test-id="settingTitle"]/parent::*[@role="dialog"]'
+settings drawer title: '[test-id=settingTitle]'
+settings dropdowns: '[test-id*=SelectDefault]'
```

Comment:
question: Is the `*` shorthand or something?

Reply:
* is wildcard, it say that find an element which has a tag name as *(any) with test-id="settingTitle